src/content/guides/limiting-pod-communication-with-network-policies/index.md

```diff
@@ -11,54 +11,130 @@ tags = ["tutorial"]
+  - from:
+    - ipBlock:
+        cidr: 172.17.0.0/16
+    - namespaceSelector:
+        matchLabels:
+          name: namespace-that-can-talk-to-my-app
+    - podSelector:
+        matchLabels:
+          app: pod-that-can-talk-to-my-app
+    ports:
+    - protocol: TCP
+      port: 6379
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 10.0.0.0/24
+    - namespaceSelector:
+        matchLabels:
+          name: namespace-my-app-can-talk-to
+    - podSelector:
+        matchLabels:
+          app: pod-my-app-can-talk-to
+    ports:
+    - protocol: TCP
+      port: 5978
 ```
 
-Applying the above policy with 
+## Default policies
 
-```nohighlight
-kubectl -n <namespace> apply -f default-deny.yaml 
+You can create default policies for a namespace by creating a NetworkPolicy that select all Pods as follows:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
 ```
 
-will result in all traffic to all Pods in the Namespace to be denied.
+__Warning__: By default Giant Swarm clusters, from version `11.0.0`, contains this default policy for sensitive namespaces like `giantswarm` and `kube-system`. To communicate with any pods in that namespace you need to explicitly create a Network Policy that allows it.
 
 Note that the namespace needs to exist before you apply the NetworkPolicy to it.
 
-### Creating selective network policies
+The default policy shown above will limit ingress and egress traffic in the namespace applied. You can restrict only for `egress` or `ingress` too. 
+
+## Applications 
+
+### Allowing specific system pod to talk with your pod
+
+As we mentioned before we harden the clusters restricting the communication with kube system and giantswarm pods. In case you need to allow that communication with a running pod in one of those namespaces you have to explicitly declare. For example:
+
+```yaml
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ksm-can-be-accessed-by-my-app
+  namespace: kube-system
+spec:
+  podSelector:
+    matchLabels:
+      app: kube-state-metrics
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: my-app-that-needs-access-to-ksm
+      ports:
+        - protocol: TCP
+          port: 10301
+```
+
+To make it more visual, this how it will look at the communication between namespaces.
 
-No matter if you set default policies or not, you can limit access to certain Pods by creating a NetworkPolicy that selects them.
+![](giant-swarm-network-policies-diagram.png)
 
-#### Allowing specific pod to pod access
+### Allowing specific pod to pod access
 
-In the following example we allow traffic to Pods labeled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
+In the following example, we allow traffic to Pods labelled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
```

Comment:
```suggestion
In the following example, we allow traffic to Pods labeled `role: backend` from Pods with the `role: frontend` label and only on TCP port 6379.
```

Both spellings are correct but labeled is preferred in 🇺🇸 which is what we aim for. https://www.grammarly.com/blog/labeled-labelled/